src/eventhubs/azext_eventhub/_params.py

```diff
@@ -0,0 +1,104 @@
+def load_arguments_namespace(self, _):
+    with self.argument_context('eventhubs') as c:
+        c.argument('resource_group_name', arg_type=resource_group_name_type)
+        c.argument('namespace_name', options_list=['--namespace-name'], help='name of the Namespace')
+
+    with self.argument_context('eventhubs namespace exists') as c:
+        c.argument('namespace_name', options_list=['--name', '-n'], help='name of the Namespace')
+
+    with self.argument_context('eventhubs namespace create') as c:
+        c.argument('namespace_name', options_list=['--name', '-n'], help='name of the Namespace')
+        c.argument('tags', options_list=['--tags', '-t'], arg_type=tags_type, help='tags for the namespace in Key value pair format')
+        c.argument('sku', options_list=['--sku-name'], arg_type=get_enum_type(['Basic', 'Standard']))
+        c.argument('location', options_list=['--location', '-l'], help='Location')
+        c.argument('skutier', options_list=['--sku-tier'], arg_type=get_enum_type(['Basic', 'Standard']))
+        c.argument('capacity', options_list=['--capacity'], type=int, help='Capacity for Sku')
+        c.argument('is_auto_inflate_enabled', options_list=['--is-auto-inflate-enabled'], type=bool, help='Value that indicates whether AutoInflate is enabled for eventhub namespace.')
+        c.argument('maximum_throughput_units', options_list=['--maximum-throughput-units'], type=int, help='Upper limit of throughput units when AutoInflate is enabled, vaule should be within 0 to 20 throughput units. ( 0 if AutoInflateEnabled = true)')
+
+    # region Namespace Get
+    for scope in ['eventhubs namespace show', 'eventhubs namespace delete']:
+        with self.argument_context(scope) as c:
+            c.argument('namespace_name', options_list=['--name', '-n'], help='name of the Namespace')
+
+    # region Namespace Authorizationrule
+    for scope in ['eventhubs namespace authorizationrule', 'eventhubs namespace authorizationrule keys list', 'eventhubs namespace authorizationrule keys renew']:
+        with self.argument_context(scope) as c:
+            c.argument('authorization_rule_name', options_list=['--name', '-n'], help='name of the Namespace AuthorizationRule')
+
+    with self.argument_context('eventhubs namespace authorizationrule create') as c:
+        c.argument('accessrights', options_list=['--access-rights'],
+                   help='Authorization rule rights of type list, allowed values are Send, Listen or Manage')
+
+    with self.argument_context('eventhubs namespace authorizationrule keys renew') as c:
+        c.argument('key_type', options_list=['--key-name'], arg_type=get_enum_type(['PrimaryKey', 'SecondaryKey']))
+
+
+# region - Eventhub Create
+def load_arguments_eventhub(self, _):
+    with self.argument_context('eventhubs eventhub') as c:
+        c.argument('event_hub_name', options_list=['--name', '-n'], help='Name of Eventhub')
+
+    with self.argument_context('eventhubs eventhub create') as c:
```

Comment:
Need to add an entry for `--capture-size-limit-bytes`

Reply:
renamed --capture-size-bytes to --capture-size-limit-bytes